src/test/java/com/github/couchmove/repository/CouchbaseRepositoryIT.java

```diff
@@ -146,6 +148,13 @@ public void should_import_fts_index() throws IOException {
 
         // Then it should be created
         assertThat(repository.isFtsIndexExists(TEST)).isTrue();
+
+        // Define expected params
+        String params = "{\"doc_config\":{\"docid_prefix_delim\":\"\",\"docid_regexp\":\"\",\"mode\":\"type_field\",\"type_field\":\"_class\"},\"mapping\":{\"analysis\":{},\"default_analyzer\":\"standard\",\"default_datetime_parser\":\"dateTimeOptional\",\"default_field\":\"_all\",\"default_mapping\":{\"dynamic\":true,\"enabled\":false},\"default_type\":\"_default\",\"docvalues_dynamic\":true,\"index_dynamic\":true,\"store_dynamic\":false,\"type_field\":\"_type\",\"types\":{\"testType\":{\"dynamic\":true,\"enabled\":true}}},\"store\":{\"indexType\":\"scorch\",\"segmentVersion\":15}}";
+        Map<String, Object> expectedParams = new ObjectMapper().readValue(params, Map.class);
+
+        // Ensure params is created as specified
+        assertThat(repository.getFtsIndexParams(TEST)).isEqualTo(expectedParams);
```

Comment:
As stated earlier, you can get the params with the new Optional method `getFtsIndex()`, and you can also access `params` in `ftsIndex` json content:
```java
SearchIndex searchIndex = ((CouchbaseRepositoryImpl<?>) repository).getFtsIndex(TEST).get();
assertThat(searchIndex.params()).isEqualTo(ftsIndexMap.get("params"));
```